lib/backbone.marionette.esm.js

```diff
@@ -1139,9 +1697,14 @@ var ViewMixin = {
     }
   }
 };
+extend$1(ViewMixin, BehaviorsMixin, CommonMixin, DelegateEntityEventsMixin, TemplateRenderMixin, UIMixin, ViewEventsMixin);
 
-_.extend(ViewMixin, BehaviorsMixin, CommonMixin, DelegateEntityEventsMixin, TemplateRenderMixin, TriggersMixin, UIMixin);
-
+function isView(view) {
```

Comment:
New way of determining what a view is.  Rather than depending on prototypal inheritance, anything that supports the API used by Marionette will be attempted.